src/YOLO/index.test.js

```diff
@@ -0,0 +1,31 @@
+/* eslint new-cap: 0 */
+
+import YOLO from './index';
+
+async function getRobin() {
+  const img = new Image();
+  img.crossOrigin = '';
+  img.src = 'https://ml5js.org/docs/assets/img/bird.jpg';
+  await new Promise((resolve) => { img.onload = resolve; });
+  return img;
+}
+
+describe('YOLO', () => {
+  let yolo;
+
+  beforeEach(async () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
```

Comment:
cc @cvalenzuela you may want to mock the actual fetch call in your unit tests so your tests are hermetic

Reply:
can you expand a bit on this? I'm not sure what hermetic means in the context.